testsuite/features/step_definitions/cobbler_steps.rb

```diff
@@ -254,15 +254,18 @@
 end
 
 Then(/^the local logs for Cobbler should not contain errors$/) do
+  node = get_target('server')
   cobbler_log_file = '/var/log/cobbler/cobbler_debug.log'
   local_file = '/tmp/cobbler_debug.log'
```

Comment:
Are we sure /tmp/ inside the container runs on a Volume with enough disk space?

Reply:
no... would you prefer me to cap the file to a given number of lines?

Reply:
or we could use /var for the backup

Reply:
No worries, all good.

Reply:
ok - we can still rectify later on

Reply:
```
uyuni-server:/ # ls -lh /var/log/cobbler/cobbler.log 
-rw-r--r--. 1 root root 14M Jun  7 10:40 /var/log/cobbler/cobbler.log
```